webpack.config.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = {
+    entry: {
+        app: './src/js/scripts.js'
+    },
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'public/dist/js')
+    },
+    devtool: 'none',
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /node_modules/,
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: [
+                                [
+                                    '@babel/preset-env',
+                                    {
+                                        targets:
+                                            'last 1 version, ie >= 11, Safari >= 12, > 0.25%, not dead'
```

Comment:
Does this need anything added for iOS / Android?

https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in